app/js/helpers/apiHelper.js

```diff
@@ -1,5 +1,8 @@
 import fetchPolyfill from 'whatwg-fetch';
 
+const contextPath = location.href.split('/')[3];
+const BASE_URL = `/${contextPath}/ws/rest/v1/`;
```

Comment:
Ok the value of the context path is set from the address the user is visiting. The `location.href` in line 2 is a javascript inbuilt way of getting the web url that the user is on. So the contextPath is set based on the address of the cohortBuilder module that the user is accessing